CONTRIBUTING.md

```diff
@@ -46,16 +46,16 @@ Should you choose to work on a new issue or an existing issue from <https://gith
 
 - Make the required changes to the component.
 - Write integrations tests and unit tests.
-- Check for all lint errors, if there are any fix it.
+- Check for all lint errors, if there are any fix them.
 - Run tests for all the components.
 - If the component you modify is used in another component make sure you update the tests for that component as well.
-- Raise a Pull Requesst to the Crayons Repository.
+- Raise a Pull Request to the Crayons Repository.
 
 ### Commit the changes to Github
 
-Crayons repository uses github pre commit hooks, follow the below steps to commit our changes to github
+Crayons repository uses github pre commit hooks, follow the below steps to commit your changes to github
```

Comment:
Could we modify this as follows?

The Crayons repository uses GitHub’s pre-commit hooks. To commit your changes,